io/disk/ssd/nvmetest.py

```diff
@@ -447,6 +457,13 @@ def test_create_namespaces(self):
         ns_count = self.params.get('namespace_count', default=1)
         device = self.device.split("/")[-1]
         nvme.create_namespaces(device, ns_count)
+        if self.shared:
+            nvme.delete_all_ns(device)
+            self.log.info(self.shared)
+            nvme.create_namespaces(device,
+                                   ns_count,
+                                   shared_ns=self.shared)
+            nvme.delete_all_ns(device)
```

Comment:
why delete all again after creation

Reply:
I followed same approach which was defined earlier in order not the break the execution flow.